bootstrapper/initproto/init.proto

```diff
@@ -25,6 +25,7 @@ message InitRequest {
   bool conformance_mode = 14;
   repeated KubernetesComponent kubernetes_components = 15;
   bytes init_secret = 16;
+  repeated bytes idkeydigests = 17;
```

Comment:
Could we send this value through the helm charts instead? Could be set in [loader.go](https://github.com/edgelesssys/constellation/blob/749acdd55d0266d6ba5bdd5b5d20d038009a956e/cli/internal/helm/loader.go#L406). This would prevent changing the init API and would be more in line with our current approach of transporting as many values as possible through the helm release values.

Reply:
Good idea!